.github/workflows/lint.yaml

```diff
@@ -1,22 +1,16 @@
 name: lint
 
 on:
-  - push
-  - pull_request
+  push:
+    branches:
+      - main
+  pull_request:
 
 jobs:
-  lint:
+  call-workflow:
+    uses: lsst/rubin_workflows/.github/workflows/lint.yaml@main
+  ruff:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-
-      - name: Set up Python
-        uses: actions/setup-python@v5
-        with:
-          python-version: '3.11'
-
-      - name: Install
-        run: pip install -r <(curl https://raw.githubusercontent.com/lsst/linting/main/requirements.txt)
-
-      - name: Run linter
-        run: flake8
+      - uses: chartboost/ruff-action@v1
```

Comment:
I see you fixed the ruff check. In future you probably want something more than the default ruff config. We strongly recommend ruff check over flake8 in general (and SQuaRE use it in their services) because it's much faster and has many more lint rules that can be enabled.